server/core/Cargo.toml

```
@@ -17,6 +17,7 @@ doctest = false
 
 [features]
 ui_htmx = []
+ui_admin = []
```

Why's this a feature in addition to the `ui_htmx` feature?

---

I thought that was what William requested here https://github.com/kanidm/kanidm/pull/3019#discussion_r1774305484. Perhaps to ship it in release while hiding it from default view because the hiding wasn't implemented yet

---

I'm okay with this as the feature gate for now.